### ISSUE TITLE
Dota Plus changed Predicted Win Streak 

### ISSUE THREAD

**Issue by cypherrage** [ValveSoftware/Dota2-Gameplay#20881]
### Description

I had 6 predicted wins in a row, then I subscribed to Dota Plus, then I had 2 predicted wins in a row.

In this match 7890885253 I had 6 predicted wins. I did not predict during this game (loss).

I subscribed to Dota Plus at 5:44pm Mountain Time.

In this match 7891080038 after I subscribed, I had 2 predicted wins.

**Comment by cypherrage** [comment]
My guess is that there are different counters for the legacy Win Prediction (which I have) and the Dota Plus win prediction.

**Comment by Jordan11599** [comment]
You are correct @cypherrage, if you check your inventory you should see 2 different Win Prediction items, but can only equip one
